### ISSUE TITLE
Can't create attributes

### ISSUE THREAD

**Issue by nelakay** [woocommerce/woocommerce#26165]
I am creating an e-commerce website with WooCommerce, and I ran into an issue with variable products. I don't seem to be able to create attributes. 
I see the option to create them, and I am able to change the parameters to do so. When I click "Add Attribute" the filled information disappears, but no attribute is created. 

**To Reproduce**
1. Go to 'Products'
2. Click on 'Attributes'
3. Fill in Name and Type, enable archive.
4. Click on 'Add Attribute'
5. Information disappears from the form on the left.
6. List of Attributes on the right is still empty.

**Expected behavior**
On the left side of the screen is a list, where the created attributes should appear. They do not. 

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://pharmly.co
Site address (URL): https://pharmly.co
WC Version: 4.0.1
REST API Version: ✔ 1.0.7
WC Blocks Version: ✔ 2.5.14
Action Scheduler Version: ✔ 3.1.4
WC Admin Version: ✔ 1.0.3
Log Directory Writable: ✔
WP Version: 5.4
WP Multisite: –
WP Memory Limit: 768 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.3.16
PHP Post Max Size: 128 MB
PHP Time Limit: 120
PHP Max Input Vars: 3000
cURL Version: 7.66.0
OpenSSL/1.1.1d-fips

SUHOSIN Installed: –
MySQL Version: 5.7.25-28-log
Max Upload Size: 128 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 4.0.1
WC Database Prefix: pwn_
Total Database Size: 4.32MB
Database Data Size: 2.90MB
Database Index Size: 1.42MB
pwn_woocommerce_sessions: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
pwn_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
pwn_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
pwn_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
pwn_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
pwn_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
pwn_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
pwn_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
pwn_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
pwn_actionscheduler_actions: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
pwn_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
pwn_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
pwn_actionscheduler_logs: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_comments: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
pwn_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
pwn_options: Data: 2.05MB + Index: 0.08MB + Engine InnoDB
pwn_postmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_posts: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
pwn_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
pwn_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_usermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
pwn_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
pwn_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
pwn_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
pwn_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
pwn_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
pwn_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
pwn_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
pwn_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
pwn_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 4
page: 6
post: 2
product: 4
revision: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (17) ###

Ocean Extra: by OceanWP – 1.6.2
Livemesh Addons for Elementor: by Livemesh – 2.9.9
Elementor Pro: by Elementor.com – 2.9.2
Elementor: by Elementor.com – 2.9.7
ElementsKit Lite: by Wpmet – 1.5.0
MyCryptoCheckout: by edward_plainview – 2.78
Ocean Product Sharing: by OceanWP – 1.1.0
Ocean Social Sharing: by OceanWP – 1.1.0
Premium Addons for Elementor: by Leap13 – 3.12.1
SG Optimizer: by SiteGround – 5.4.6
Sticky Header OceanWP: by Oren Hahiashvili – 1.0.5
Variation Swatches for WooCommerce: by Emran Ahmed – 1.0.80
WOOCS - WooCommerce Currency Switcher: by realmag777 – 1.3.1
WooCommerce PayPal Checkout Gateway: by WooCommerce – 1.6.20 – Not tested with the active version of WooCommerce
WooCommerce Stripe Gateway: by WooCommerce – 4.3.3
WooCommerce Services: by Automattic – 1.23.0
WooCommerce: by Automattic – 4.0.1

### Inactive Plugins (0) ###


### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #8 - /shop/
Cart: #9 - /cart/
Checkout: #10 - /checkout/
My account: #11 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: OceanWP
Version: 1.8.2
Author URL: https://oceanwp.org/about-me/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: oceanwp/woocommerce/cart/mini-cart.php
oceanwp/woocommerce/content-single-product.php
oceanwp/woocommerce/loop/loop-start.php
oceanwp/woocommerce/single-product/title.php


### Action Scheduler ###

Complete: 2
Oldest: 2020-04-09 15:20:12 +0000
Newest: 2020-04-09 15:46:54 +0000

`
```
</details>

**Comment by juliaamosova** [comment]
Hi @nelakay,

Thank you for submitting the issue. However, I can’t reproduce the issue you reported using the steps you provided. Everything is working as expected on my end. 

I made a video to show that I am able to create attributes: https://recordit.co/XdCF4EZXc3. 

You indicated that you tested your site for themes and plugins conflict, however, I always like to confirm. Have you in fact done it? 

I see that you have several plugins installed and that you don’t use one of our default themes which could cause an issue. Please test your site for theme and plugins conflict. To do that, you’d need to deactivate all plugins except for WooCommerce and switch the default theme such as [Storefront](https://en-gb.wordpress.org/themes/storefront/). Then test again.

If the issue is resolved with the default theme and all plugins deactivated, it means that one of your plugins or a theme is causing the issue. You will then need to enable it one by one and test every time you do that in order to figure out which plugin is causing the issue.

**Comment by nelakay** [comment]
Hello,

Thank you for your response.

Yes, I in fact did it. I changed the theme to Storefront, and tested
without an plugins, but still nothing.
In fact, just to be sure, I just did it again now. No result. I am still
unable to create attributes.

**Comment by nelakay** [comment]
Hello?

**Comment by juliaamosova** [comment]
@nelakay thanks for additional testing and follow-up. I can recommend going over the doc below that explains the different steps you can take to fix the issue. It sounds like it is hosting/server related in your case:

https://docs.woocommerce.com/document/problems-with-large-amounts-of-data-not-saving-variations-rates-etc/

**Comment by nelakay** [comment]
Hello again,

I fixed the problem. The title I was using to try to create attributes was
"Type" I tried "Option" instead, and it worked.

However, now I have a new issue. I am creating a menu under the header of
my website. This header is fed with a number of product categories. Now the
header has two carets in each section (only the ones that have a
subsection, see screenshot attached). I was troubleshooting, and the menu
disappears completely as soon as I switch off the Woocommerce plugin. Is
there anything I am doing wrong?

Thank you.

**Comment by juliaamosova** [comment]
@nelakay I am glad to hear your original issue related to you not being able to create attributes is resolved. 

>However, now I have a new issue. I am creating a menu under the header of
my website. This header is fed with a number of product categories. Now the
header has two carets in each section (only the ones that have a
subsection, see screenshot attached). I was troubleshooting, and the menu
disappears completely as soon as I switch off the Woocommerce plugin. Is
there anything I am doing wrong?

While our goal is to address all the issues reported in this repository, GitHub should be treated as a place to report confirmed bugs only. **This is not a place to receive technical support or ask questions about building your site.** 

Your question looks like a support request which may or may not reveal a bug once proper troubleshooting is done. In order to confirm the bug, please follow one of the steps below:

- Review [WooCommerce Self-Service Guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/) to see if the solutions listed there apply to your case;
- If you are a paying customer of WooCommerce, contact WooCommerce support by [opening a ticket](https://woocommerce.com/contact-us/);
- Make a post on [WooCommerce community forum](https://wordpress.org/support/plugin/woocommerce/)

If you confirm the bug, please provide us with clear steps to reproduce it.

I am closing this issue for now as it seems to be a support request and not a bug. If I missed something, please leave a comment and we will take a second look.
